git-standup

```diff
@@ -41,77 +41,83 @@ function colored() {
   NORMAL=$(tput sgr0)
 
   GIT_PRETTY_DATE="%cd"
-  if [[ $option_R ]] ; then
+  # shellcheck disable=SC2154
+  if [[ $option_R ]]; then
     GIT_PRETTY_DATE="%ad"
   fi
 
   GIT_PRETTY_FORMAT="%Cred%h%Creset - %s %Cgreen($GIT_PRETTY_DATE) %C(bold blue)<%an>%Creset"
   COLOR=always
 
-  if [[ $option_g ]] ; then
+  # shellcheck disable=SC2154
```

Comment:
Can we have the [global directive](https://github.com/koalaman/shellcheck/wiki/Ignore#ignoring-all-instances-in-a-file-044) instead of repetitive ignore?

Reply:
Personally I do not like to mute globally such a warning "[var is referenced but not assigned](https://github.com/koalaman/shellcheck/wiki/SC2154)" because such warning make you awareness about using variable not declared/assigned